startup/39-original_suitcase.py

```diff
@@ -28,14 +26,45 @@ def conv_to_list(d):
     return d1 
 
 def update_res_path(res_path, replace_res_path={}):
-    #res_path = res["resource_path"]
     for rp1,rp2 in replace_res_path.items():
         print("updating resource path ...")
         if rp1 in res_path:
             res_path = res_path.replace(rp1, rp2)  
     return res_path
 
-def hdf5_export(headers, filename,
+def locate_h5_resource(res, replace_res_path, debug=False):
+    """ this is intended to move h5 file created by Pilatus
+        these files are originally saved on PPU RAMDISK, but should be moved to the IOC data directory
+        this function will look for the file at the original location, and relocate the file first if it is there
+        and return the h5 dataset
+    """
+    fn_orig = res["root"]+res["resource_path"]
+    fn = update_res_path(fn_orig, replace_res_path)
+    if debug:
+        print(f"resource locations: {fn_orig} -> {fn}")
+    
+    if os.path.exists(fn_orig):
+        if os.path.exists(fn):
+            print(f"both {fn} and {fn_orig} exist, resolve the conflict manually first ..." )
+            raise Exception
+        fdir = os.path.dirname(fn)
+        if not os.path.exists(fdir):
+            makedirs(fdir, mode=0o2775)
+        if debug:
+            print(f"copying {fn_orig} to {fdir}")
+        tfn = fn+"_partial"
+        shutil.copy(fn_orig, tfn)
+        os.rename(tfn, fn)
+        os.remove(fn_orig)
+    elif not os.path.exists(fn):
+        print(f"could not location the resource at either {fn} or {fn_orig} ...")
```

Comment:
```suggestion
        print(f"could not locate the resource at either {fn} or {fn_orig} ...")
```

Reply:
fixed in https://github.com/NSLS-II-LIX/profile_collection/pull/13/commits/c15dddc5150db915a5703155dbfa6243e4c6a666